dodo.py

```diff
@@ -643,17 +649,18 @@ def task_tar():
         '--exclude=__pycache__',
         '--exclude=*.pyc',
         '--exclude=.env',
-        '--exclude=.git',
+        '--exclude=sub/tests',
+        '--exclude=hub/tests',
+        '--exclude=shared/tests'
     ])
     for src in SRCS:
         ## it is important to note that this is required to keep the tarballs from
         ## genereating different checksums and therefore different layers in docker
-        cmd = f'cd {CFG.REPO_ROOT}/src/{src} && echo "$(git status -s)" > {CFG.REVISION} && tar cvh {excludes} . | gzip -n > {CFG.SRCTAR} && rm {CFG.REVISION}'
+        cmd = f'cd {CFG.REPO_ROOT} && cp -rf .git {CFG.REPO_ROOT}/src/{src} && cd {CFG.REPO_ROOT}/src/{src} && echo "$(git status -s)" > {CFG.REVISION} && tar cvh {excludes} . | gzip -n > {CFG.SRCTAR} && rm {CFG.REVISION}'
```

Comment:
what is this? why?

Reply:
Don't you want the version/branch/etc from `.git`